pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -84,6 +84,28 @@ def work(self):
                                 else:
                                     pokeball = 0 # player doesn't have any of pokeballs, great balls or ultra balls
                             
+                            ## Use berry to increase success chance.
+                            berry_id = 701 # @ TODO: use better berries if possible
+                            berries_count = self.bot.item_inventory_count(berry_id)
+                            if(catch_rate[pokeball-1] < 0.5 and berries_count > 0): # and potion is in stock
+                                success_percentage = '{0:.2f}'.format(catch_rate[pokeball-1]*100)
+                                logger.log('[x] Catch Rate is low ({}%). Throwing {}... ({} left!)'.format(success_percentage,self.item_list[str(pokeball)],berries_count-1))
```

Comment:
`success_percentage,self.item_list[str(pokeball)],berries_count-1)`
[x] Catch Rate is low ({}%). Throwing **Pokeball**...
Typo